src/action/autopilot.js

```diff
@@ -48,6 +55,69 @@ class AtplAction {
       this._notification.display({ msg: 'Error toggling autopilot', err });
     }
   }
+
+  /**
+   * Update autopilot's channel scores so it knows whom to open channels with.
+   * @return {Promise<undefined>}
+   */
+  async updateAutopilotScores() {
+    try {
+      const nodeScores = await this.fetchBosScores();
+      await this._grpc.sendAutopilotCommand('setScores', {
+        heuristic: 'externalscore',
+        scores: nodeScores,
+      });
+    } catch (err) {
+      log.error('Updating autopilot scores failed', err);
+    }
+  }
+
+  /**
+   * Fetch bos scores, persist them, and return them in a list.
+   * @return {Object}
+   */
+  async fetchBosScores() {
+    const networkStr =
+      (await this._getNetwork()) === 'testnet' ? 'testnet' : '';
+    try {
+      const uri = `https://nodes.lightning.computer/availability/btc${networkStr}.json`;
```

Comment:
We should move the base url `https://nodes.lightning.computer` to config.js

Reply:
Oh and it's
mainnet: https://nodes.lightning.computer/availability/btc.json
testnet: https://nodes.lightning.computer/availability/btctestnet.json